### ISSUE TITLE
Colons in date format string enclosed in single quotes treated as parameters.

### ISSUE THREAD

**Issue by ericxinzhang** [pihvi/yesql#13]
Code snippet:

```
const named = require('yesql').pg;
...
const sql = `select name, value, to_char(created_at,  'YYYY-MM-DD HH24:MI:SS') from table1 where created_at > :from and created_at <= :to;`;
return named(sql)({from: new Date(0), to: new Date()});
```

Got following error:
```
Error: Missing value for statement.
MI not provided for statement:
...
```

**Comment by pihvi** [comment]
Thanks for the report! I'll start working on this asap.

**Comment by pihvi** [comment]
This is now supported in just released version 4.0.0.

Please report if it's not working as expected.

**Comment by ericxinzhang** [comment]
> This is now supported in just released version 4.0.0.
> 
> Please report if it's not working as expected.

That's awesome! Thanks for quick response!
